openedx/frontend-app-learning
# Multiple not enrolled alert fixed

Fix multiple enrolled alert.

**JIRA tickets**: 
* https://openedx.atlassian.net/browse/TNL-7249
* https://openedx.atlassian.net/browse/OSPR-4741

**Screenshots**: 
|  Before | After |
| --------- | ------- |
| ![image](https://user-images.githubusercontent.com/3592998/85276371-1dbbd100-b49b-11ea-85be-95dcb4acec7c.png) | ![image](https://user-images.githubusercontent.com/3592998/85276256-f36a1380-b49a-11ea-82cf-4e2e00101096.png) |


**Merge deadline**: "None"

**Testing instructions**:

1. Run [edx master devstack](https://github.com/edx/devstack).
2. Run [frontend-app-learning](https://github.com/edx/frontend-app-learning).
3. Visit http://localhost:18000 and login as staff.
4. Visit http://localhost:2000/course/course-v1:edX+DemoX+Demo_Course/outline.
5. Click on `Introduction` section and then `Demo Course Overview` link.
6. You should see only a single alert.
7. Press the back button of your browser.
8. Again perform `step 5`.
9. You should see only a single alert. If you see more than one, the issue is not fixed otherwise fixed.


**Reviewers**
